### ISSUE TITLE
Cifar 10 numbers reported use data augmentation

### ISSUE THREAD

**Issue by FredericGodin** [titu1994/DenseNet#8]
Could it be that the accuracy reported for the CIFAR10 dataset is obtained using data augmentation?
This is somehow misleading because the accuracy reported in the paper doesn't use it.

**Comment by titu1994** [comment]
@FredericGodin The paper does use standard data augmentation techniques. See Table 2, and its description underneath. "+" next to a dataset means it went through standard data augmentation, which is what I am comparing my results against.

**Comment by FredericGodin** [comment]
Let me rephrase that. I meant that the data augmentation used is different from the one mentioned in the paper. 

```
generator = ImageDataGenerator(rotation_range=15,
                               width_shift_range=5./32,
                               height_shift_range=5./32)
```

The standard data augmentation is mirroring and using a random crop I believe?
You use rotation?

From the Lua code:

```
function CifarDataset:preprocess()
   if self.split == 'train' then
      return t.Compose{
         t.ColorNormalize(meanstd),
         t.HorizontalFlip(0.5),
         t.RandomCrop(32, 4),
      }
   elseif self.split == 'val' then
      return t.ColorNormalize(meanstd)
   else
      error('invalid split: ' .. self.split)
   end
end
```

**Comment by titu1994** [comment]
Oh that's what you meant. Yes, I did not use horizontal mirroring even though it was there because it was causing poor results compared to not using it. 

Center cropping isn't available yet in image preprocessing so I could not implement that. 

There are several other papers which use width and height shift as standard augmentation for Cifar 10 and 100, though they also suggest horizontal mirroring.
